scripts/ci/jenkins_pipeline_csi

```diff
@@ -1,4 +1,11 @@
 pipeline {
+    environment {
+        registryCredential = 'csi_w3_user'
+        artifactory_path = 'build/reports/images_url'
```

Comment:
why not `registry_path`?

Reply:
because it's the path that the artifactory phase uses

Reply:
but here we use artifactory only as a docker registry, and this pipeline code should work with non-artifactory registries as well.
so I'm not sure I follow. could you please clarify why `artifactory` is preferred over `registry`?

Reply:
I didn't mean to our local artifactory, I meant to the artifactory plugin that shows us at the end of the run the docker images that the pipeline created

Reply:
you mean [this plugin](https://plugins.jenkins.io/artifactory/)? what does it have to do with [archiveArtifacts](https://www.jenkins.io/doc/pipeline/steps/core/) (which you're [not using](https://github.com/IBM/ibm-block-csi-driver/blob/ccc95bf4661a7b421b19e0b395faee03a33ec0ea/scripts/ci/jenkins_pipeline_csi#L86) with `artifactory_path` anyway)?

Reply:
It's supposed to be used by the archiveArtifacts, but it wasn't in the scope of my task, I will open a ticket to go through the pipelines and to fix some issues that in there (I saw a few).

Reply:
this does not answer my questions, the lack of `artifactory_path` usage by `archiveArtifacts` was only a side note.
could you please clarify what does `build/reports/images_url` has to do with `artifactory`?

Reply:
removed it, done